optimum/neuron/utils/cache_utils.py

```diff
@@ -208,6 +211,12 @@ def get_neuron_cache_path() -> Optional[Path]:
         else:
             path = Path("/var/tmp")
 
+        # TODO: is that correct?
+        # neuronxcc_version = get_neuronxcc_version()
+        # if version.parse(neuronxcc_version) < version.parse(_NEW_CACHE_NAMING_CONVENTION_NEURONXCC_VERSION):
```

Comment:
If neuronxcc_version < compiled version, it's not going to work. The inversed is compatible according to Annapurna but I seems encountered issue with backward compatibility either.